weave-js/src/components/Panel2/ChildPanel.tsx

```diff
@@ -758,6 +758,12 @@ const NEW_INSPECTOR_IMPLEMENTED_FOR = new Set([
   `Sections`,
 ]);
 
+// Return true if maybeAncestor is an ancestor of path.
+// Returns false for self and other non-ancestors.
+const isAncestor = (path: string, maybeAncestor: string): boolean => {
+  return path.startsWith(maybeAncestor + '.');
```

Comment:
should this be `.includes`? Seems like this is determining if it is the root ancestor, rather than any arbitrary ancestor.